generated/google_cloud_ai_generativelanguage_v1beta/example/streamed_response.dart

```diff
@@ -29,9 +29,9 @@ void main() async {
   );
 
   await for (final result in service.streamGenerateContent(request)) {
-    final parts = result.candidates?[0].content?.parts;
+    final parts = result.candidates.first.content?.parts;
     if (parts != null) {
-      parts.forEach((p) => stdout.write(p.text ?? ''));
+      parts.forEach((p) => stdout.write(p.text));
```

Comment:
```suggestion
      parts.forEach((p) => stdout.write(p.text!));
```

I think? Could you rerun these examples.